src/Microsoft.PowerShell.Commands.Management/commands/management/WebServiceProxy.cs

```diff
@@ -273,7 +274,6 @@ protected override void BeginProcessing()
         #region private
 
         private static ulong s_sequenceNumber = 1;
-        private static object s_sequenceNumberLock = new object();
```

Comment:
Actually, I just found that we don't even compile the `WebServiceProxy.cs` (see `Microsoft.PowerShell.Commands.Management.csproj` for the `<compile remove=` entries). So, please revert the changes in this file.